shared/lib/accounts/snaps.ts

```diff
@@ -1,6 +1,22 @@
 import { SnapId } from '@metamask/snaps-sdk';
-import { BITCOIN_WALLET_SNAP_ID } from './bitcoin-wallet-snap';
+import {
+  stripSnapPrefix,
+  getLocalizedSnapManifest,
+} from '@metamask/snaps-utils';
+// eslint-disable-next-line import/no-restricted-paths
+import type { SnapKeyringBuilderMessenger } from '../../../app/scripts/lib/snap-keyring/types';
 import { SOLANA_WALLET_SNAP_ID } from './solana-wallet-snap';
+import { BITCOIN_WALLET_SNAP_ID } from './bitcoin-wallet-snap';
+
+export const PREINSTALLED_SNAPS = [
+  'npm:@metamask/message-signing-snap',
+  'npm:@metamask/ens-resolver-snap',
+  'npm:@metamask/institutional-wallet-snap',
+  'npm:@metamask/account-watcher',
+  'npm:@metamask/preinstalled-example-snap',
+  'npm:@metamask/bitcoin-wallet-snap',
+  'npm:@metamask/solana-wallet-snap',
+];
```

Comment:
We should probably move those in a new file like `shared/lib/snaps.ts` (since some Snaps are not "Account Management Snaps").

Reply:
actually, maybe we could use a folder in between, so this would become owned by the Snaps team? 🤔